dnf-behave-tests/dnf/comps-same-id-group-and-env.feature

```diff
@@ -0,0 +1,226 @@
+        | group-remove  | Same id group                   |
+        | remove        | C-mandatory-1.0-1.x86_64        |
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/groups/same-id.xml" does not exist
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/environments/same-id.xml" does not exist
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/same-id.xml" does not exist
+
+
+Scenario: Upgrade a group with the same id as an environment
+ Given I use repository "comps-same-id-1"
+   And I execute dnf with args "install @same-id"
+   And I use repository "comps-same-id-2"
+  When I execute dnf with args "group upgrade same-id"
+  Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                         |
+        | group-upgrade | Same id group                   |
+        | install-group | D-mandatory-1.0-1.x86_64        |
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/groups/same-id.xml" exists
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/environments/same-id.xml" exists
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/same-id.xml" does not exist
+
+
+Scenario: Upgrade an environment with the same id as a group
+ Given I use repository "comps-same-id-1"
+   And I execute dnf with args "install @same-id"
+   And I use repository "comps-same-id-2"
+  When I execute dnf with args "environment upgrade same-id"
+  Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                         |
+        | env-upgrade   | Same id env                     |
+        | group-upgrade | AB-group                        |
+        | install-group | B-mandatory-1.0-1.x86_64        |
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/groups/same-id.xml" exists
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/environments/same-id.xml" exists
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/same-id.xml" does not exist
+
+
+Scenario: Upgrade both group and environment with the same id
+ Given I use repository "comps-same-id-1"
+   And I execute dnf with args "install @same-id"
+   And I use repository "comps-same-id-2"
+  When I execute dnf with args "upgrade @same-id"
+  Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                         |
+        | env-upgrade   | Same id env                     |
+        | group-upgrade | AB-group                        |
+        | install-group | B-mandatory-1.0-1.x86_64        |
+        | group-upgrade | Same id group                   |
+        | install-group | D-mandatory-1.0-1.x86_64        |
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/groups/same-id.xml" exists
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/environments/same-id.xml" exists
+    And file "/usr/lib/sysimage/libdnf5/comps_groups/same-id.xml" does not exist
+
+
+Scenario: Install a group with the same id and name as an environment
```

Comment:
All the 6 tests with "same-id-and-name" identifier have exactly the same output (sans the names) as the corresponding subset of 6 tests with "same-id". What's their point?